backend/init_data/skills/document/document_tool.py

```
@@ -0,0 +1,272 @@
+    def _run(
+        self,
+        document_type: str,
+        run_manager: Optional[AsyncCallbackManagerForToolRun] = None,
+    ) -> str:
+        """Synchronous run - not implemented."""
+        raise NotImplementedError("LoadDocumentSkillTool only supports async execution")
+
+    async def _arun(
+        self,
+        document_type: str,
+        run_manager: Optional[AsyncCallbackManagerForToolRun] = None,
```

Ruff (0.14.13)

Unused method argument: `run_manager`

(ARG002)